test/dropbox/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+
+services:
+  openapi_mock:
+    image: muonsoft/openapi-mock
+    environment:
+      OPENAPI_MOCK_USE_EXAMPLES: if_present
+      OPENAPI_MOCK_SPECIFICATION_URL: https://raw.githubusercontent.com/offen/docker-volume-backup/2ea59a649be79d6cb22b8b5ad29e24377d4ab9d4/test/dropbox/user_v2.yaml
+    ports:
+      - 8080:8080
+
+  backup:
+    image: offen/docker-volume-backup:${TEST_VERSION:-canary}
+    hostname: hostnametoken
+    depends_on:
+      - openapi_mock
+    restart: always
+    environment:
+      BACKUP_FILENAME_EXPAND: 'true'
+      BACKUP_FILENAME: test-$$HOSTNAME.tar.gz
```

Comment:
We don't really test the file being uploaded anywhere here, so I would remove these values. When removed, the `hostname` can go as well.

Reply:
It does actually look for files with the prefix 'test' during pruning, the ListFolders command will return a (predefined) response with such a file. So it will be used. Just no modifications to the fs will be done (delete/upload).